packages/pg/lib/result.js

```diff
@@ -9,9 +9,10 @@ const matchRegexp = /^([A-Za-z]+)(?: (\d+))?(?: (\d+))?/
 // passed as second argument to provided callback
 class Result {
   constructor(rowMode, types) {
-    this.command = null
-    this.rowCount = null
-    this.oid = null
+    this._command = undefined
```

Comment:
Speaking of esoteric backwards compatibility needs: since the minimum Node version is 16, private class fields are fully supported. Could start using those instead of an underscore prefix for everything, to make future backwards compatibility easier to manage (since there’s no way user code can come to rely on private fields, whether it’s through direct access or stringification).